docs/guide/custom-snipptes.md

```diff
@@ -78,6 +83,97 @@ now the page-object contains our new function.:
   }
 ```
 
+* ### CustomSnippets For FillForm
+
+In this type you can extend the [FillForm](./add-meethodes.md#addfillform) methode, to let you fill out your custom form elements.
+The fillForm methode requires a data parameter. You can define the type in the custom snippet.
```

Comment:
The fillForm methode automatically has a "data" input parameter. You can define its properties in the custom snippet.

Stimmt das so?
Falls ja finde ich es so noch verständlicher.